src/box/lua/load_cfg.lua

```diff
@@ -1091,7 +1120,7 @@ end
 -- log.cfg of log module for example).
 local function update_cfg(option, value)
     if box_is_configured then
-        rawset(box.cfg, option, value)
+        raw_cfg[option] = value
```

Comment:
The same as for the three similar hunks above.

Reply:
Yeah, that one is definitely required. Anyway, I see the second patch totally removes `rawset` from `load_cfg.lua`, and that's neat. Thanks!